fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/FedoraResourceImpl.java

```diff
@@ -493,6 +494,8 @@ public void replaceProperties(final IdentifierConverter<Resource, FedoraResource
         try {
             new RdfRemover(idTranslator, getSession(), replacementStream
                     .withThisContext(differencer)).consume();
+        } catch (final ConstraintViolationException e) {
```

Comment:
Yeah, that's a very good point. I think you've got it right now. Most folks are not going to throw in RDF with many many violations, anyway.